string directory_name
---
sensor_msgs/PointCloud2[] pointclouds